regression/cbmc/compact-trace/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--compact-trace
+activate-multi-line-match
+^EXIT=10$
+^SIGNAL=0$
+^↳ main\.c:10 main\(\)\n  12: x=1 (.*)$
```

Comment:
The parentheses around `.*` don't add value.

Reply:
Fixed

Reply:
Fixed.